src/boot/bootctl-uki.c

```diff
@@ -141,15 +141,52 @@ static int read_pe_section(
         return 0;
 }
 
-static int inspect_osrel(const void *osrel, size_t osrel_size) {
-        _cleanup_fclose_ FILE *s = NULL;
+static int uki_read_string(
```

Comment:
Maybe uki_read_section_as_string()? Current name doesn't really tell you we'll read a section as a string.

Or just `uki_read_section()`?